movies_database.sql

```diff
@@ -0,0 +1,17 @@
+CREATE DATABASE movies;
```

Comment:
This can be helpful in dev but shouldn't be committed since it isn't actually part of the project